pke/kyber/internal/common/poly_test.go

```diff
@@ -1,11 +1,18 @@
 package common
 
 import (
-	"crypto/rand"
+	cryptoRand "crypto/rand"
 	"fmt"
+	"math/rand"
```

Comment:
it would be good if we get ride from math/rand. I think mathRand must be explicitly used too.

```suggestion
	mathRand "math/rand"
```